src/components/ProfileSideNav/ProfileSideNav.css

```diff
@@ -0,0 +1,18 @@
+.ProfileSideNav a {
+  display: block;
+  padding: 10px;
+  text-decoration: none;
+}
+
+.ProfileSideNav h1 {
+  padding: 10px;
+  font-weight: bold;
+  margin: 0;
+}
+
+@media screen and (max-width: 400px) {
```

Comment:
It's not ok to have this rule here because we may use this component in other places as well where we don't want to hide it under 400px. This component is generic, so it should "hide itself" based on screen size.

The correct way to do this is to put this rule in the specific context where you need it, meaning in the specific page/parent where you want this to be hidden under 400px.  Potential options:

1. Add a className to this component in it's parent then move this rule in the parent `.css` file
2. Use tailwind responsive in the parent and a class similar with `sm:hidden` over there.

Reply:
Did you see this comment?